core/src/main/java/org/mskcc/cbio/portal/util/ValueParser.java

```diff
@@ -598,9 +598,9 @@ public String getCopyNumberStyle() {
             case Diploid:
                return "no_change";
             case HemizygouslyDeleted:
-               return "hetloss_2";
```

Comment:
Updated CSS class names. But it looks like `del_1` wasn't given a case originally, I am not sure where the corresponding element is assigned